airbyte-integrations/connectors/source-oracle/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/OracleSourceComprehensiveTest.java

```diff
@@ -0,0 +1,306 @@
+            .airbyteType(JsonSchemaPrimitive.STRING)
+            .fullSourceDataType("TIMESTAMP WITH TIME ZONE")
+            .addInsertValues("to_timestamp_tz('21-FEB-2009 18:00:00 EST', 'DD-MON-YYYY HH24:MI:SS TZR')",
+                "to_timestamp_tz('21-FEB-2009 18:00:00 -5:00', 'DD-MON-YYYY HH24:MI:SS TZH:TZM')")
+            .addExpectedValues("2009-02-21 18:00:00.0 EST", "2009-02-21 18:00:00.0 -5:00")
+            .build());
+
+    DateFormat utcFormat = new SimpleDateFormat("dd-MMM-yyyy HH:mm:ss");
+    utcFormat.setTimeZone(TimeZone.getTimeZone(Calendar.getInstance().getTimeZone().getID()));
+    Date date = null;
+    try {
+      date = utcFormat.parse("21-Feb-2009 18:00:00");
+    } catch (ParseException e) {
+      e.printStackTrace();
+    }
+    DateFormat currentTFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+    currentTFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
+    String utc = currentTFormat.format(date);
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("TIMESTAMP")
+            .airbyteType(JsonSchemaPrimitive.STRING)
+            .fullSourceDataType("TIMESTAMP WITH LOCAL TIME ZONE")
+            .addInsertValues("to_timestamp_tz('21-FEB-2009 18:00:00', 'DD-MON-YYYY HH24:MI:SS')")
+            .addExpectedValues(utc + ".0 UTC")
+            .build());
+
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("INTERVAL")
+            .airbyteType(JsonSchemaPrimitive.STRING)
+            .fullSourceDataType("INTERVAL YEAR TO MONTH")
+            .addInsertValues("INTERVAL '10-2' YEAR TO MONTH", "INTERVAL '9' MONTH", "null")
+            .addExpectedValues("10-2", "0-9", null)
+            .build());
+
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("BLOB")
+            .airbyteType(JsonSchemaPrimitive.STRING)
+            .addInsertValues("utl_raw.cast_to_raw('some content here')", "null")
+            .addExpectedValues("c29tZSBjb250ZW50IGhlcmU=", null)
+            .build());
+
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("CLOB")
+            .airbyteType(JsonSchemaPrimitive.STRING)
+            .addInsertValues("utl_raw.cast_to_raw('some content here')", "null")
+            .addExpectedValues("736F6D6520636F6E74656E742068657265", null)
+            .build());
+
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("RAW")
+            .airbyteType(JsonSchemaPrimitive.STRING)
+            .fullSourceDataType("RAW(200)")
+            .addInsertValues("utl_raw.cast_to_raw('some content here')", "null")
+            .addExpectedValues("c29tZSBjb250ZW50IGhlcmU=", null)
+            .build());
+
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("LONG")
+            .airbyteType(JsonSchemaPrimitive.STRING)
+            .fullSourceDataType("LONG RAW")
+            // @TODO stream fails when reading data back
```

Comment:
Can you create a new issue to fix the implementation to handle these tests

Reply:
https://github.com/airbytehq/airbyte/issues/4307